### PR TITLE
Playwright: move currently skipped and flakey tests into Quarantine build.

### LINKED ISSUE #57866: Task: move skipped and intermittently failing tests to quarantine build
### Details

In a followup to #57619, move the categories of tests mentioned above to a quarantined build.

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by worldomonation**
#### Changes proposed in this Pull Request

This PR moves some tests into the quarantined category.

#### Testing instructions

Closes #57866.

### CODE REVIEW

File: test/e2e/specs/specs-playwright/wp-signup__free.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * @group calypso-release
+ * @group quarantined
```
Comment (dpasque) [inline test/e2e/specs/specs-playwright/wp-signup__free.ts]: I'm merging my updated PR today (unless there's big issues with it!) so I think you can leave this out for now.

Unless I totally make these specs worse, and then we can put them back in there 🙃

File: test/e2e/specs/specs-playwright/wp-signup__paid.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * @group calypso-release
+ * @group quarantined
```
Comment (dpasque) [inline test/e2e/specs/specs-playwright/wp-signup__paid.ts]: Same with this one!

**Review by dpasque — APPROVED** [review]
Sounds good! Do we want to leave the signup in to start with my new changes coming through?

File: test/e2e/specs/specs-playwright/wp-signup__free.ts (hunk 1 above)
Comment (worldomonation) [reply test/e2e/specs/specs-playwright/wp-signup__free.ts]: I was thinking we can run your updated tests in the quarantined build while we ensure it is stable, hence this change. Let's take the other approach and demote it if necessary instead.

**Pull request merged**